roles/setup-services/tasks/configure-keycloak.yml

```diff
@@ -1,244 +1,209 @@
----
+---	
```

Comment:
Please remove these extra tab spaces from this file. Ansible is complaining about it:
```
TASK [setup-services : Assign roles to users] ***************************************************
fatal: [localhost]: FAILED! => {"reason": "Syntax Error while loading YAML.\n  found character '\\t' that cannot start any token\n\nThe error appears to be in '/Users/psturc/work/aerogear/mobile-services-installer/roles/setup-services/tasks/configure-keycloak.yml': line 5, column 27, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: Get keycloak route\n                          ^ here\nThere appears to be a tab character at the start of the line.\n\nYAML does not use tabs for formatting. Tabs should be replaced with spaces.\n\nFor example:\n    - name: update tooling\n      vars:\n        version: 1.2.3\n#    ^--- there is a tab there.\n\nShould be written as:\n    - name: update tooling\n      vars:\n        version: 1.2.3\n# ^--- all spaces here.\n"}
```

Reply:
Sorry, I probably wasn't clear here. There are a lot of lines containing a tab space which ansible is not comfortable with. I'm still getting errors.
Quick fix in **vim:** `:%s/\t//g`